Server/User/redis_migrator_gui.py

```
@@ -0,0 +1,474 @@
+    def test_source_connection(self):
+        """Test source Redis connection"""
+        url = self.source_url.get().strip()
+        if not url:
+            messagebox.showerror("Error", "Please enter source Redis URL")
+            return
+            
+        try:
+            self.log(f"Testing source connection: {url}")
+            client = self.create_redis_client(url)
+            info = client.info()
+            self.log(f"✅ Source connection successful!")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)